src/web/components/Core/Accordion.tsx

```diff
@@ -0,0 +1,50 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as RadixAccordion from '@radix-ui/react-accordion';
+import { ReactNode } from 'react';
+
+import './Accordion.scss';
+
+export type AccordionProps = {
+  title: string;
+  content: ReactNode;
+  defaultExpanded: boolean;
+  isCollapsible: boolean;
```

Comment:
I thought about maybe creating a separate component to support the case where isCollapsible is false, or maybe making these props optional since some combinations don't make sense. e.g. label should only be defined when isCollapsible is false. let me know what you think

Reply:
for instance, wondering if these two should be different components or the same component with different props:

![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/764acc35-d466-4ffb-99fd-c5d6aafa69d8)

![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/b7832cb0-608e-4745-a24c-39f2fcabdc63)

Reply:
differences of note:
- border styling
- label
- collapsibility
- Behaviour of deselecting the recommendations

Reply:
For me it makes sense to be two components, there is no reason to use Accordion if the component is not Collapsible

Reply:
agreed, will do